test/catkin_package_tests/p1/foo/package.xml

```diff
@@ -14,9 +14,6 @@
   <author>John Doe</author>
   <author email="jane.doe@example.com">Jane Doe</author>
 
-  <build_depend>catkin</build_depend>
-  <build_depend version_gte="1.1" version_lt="2.0">genmsg</build_depend>
```

Comment:
I'm afraid these packages would not be available in non-ROS context, which nosetest in CI runs in. So getting rid of these. I looked around and couldn't find test cases that these removal directly affects (although there may be test cases that implicitly depend on them?).